demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -0,0 +1,3065 @@
+{
```

Comment:
<p>The csv dataset is the preprocessed version of the Pubmed data. For public demo this should be either provided in addition, or the preprocessing code made available. </p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
That's been updated now.